v0.5/translation/gnmt/tensorflow/nmt/utils/nmt_utils.py

```diff
@@ -81,20 +83,27 @@ def decode_and_evaluate(run,
             num_sentences += batch_size
             for sent_id in range(batch_size):
               for beam_id in range(num_translations_per_input):
-                translation = get_translation(
-                          nmt_outputs[beam_id],
-                         sent_id,
-                         tgt_eos=tgt_eos,
-                         subword_option=subword_option)
-                if run == 'accuracy':
-                  trans_f.write((translation + b"\n").decode("utf-8"))
+                translation.append(get_translation(
+                    nmt_outputs[beam_id],
+                    sent_id,
+                    tgt_eos=tgt_eos,
+                    subword_option=subword_option))
 
           except tf.errors.OutOfRangeError:
             utils.print_time(
                 "  done, num sentences %d, num translations per input %d" %
                 (num_sentences, num_translations_per_input), overall_start)
             break
 
+
+    if run == 'accuracy':
```

Comment:
Per the rules:
> Note: For v0.5, the same code must be run for both the accuracy and performance LoadGen modes. This means the same output should be passed in QuerySampleComplete in both modes.

So, I don't think you can take a different action based on whether you are in accuracy mode or not here.

Reply:
ok, I will make change so that it make no difference between accuracy mode and performance mode.

Reply:
I think the change I made is just moved the code to output the translation results. I didn't change the behavior of of different mode. In the original code at line 89, it already had "if run == 'accuracy':", because the model only output the translation results to a file in "accuracy" mode, not the performance mode.